spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/convert/SpannerWriteMethodCoverageTest.java

```diff
@@ -45,7 +47,18 @@ public void allKnownMappingTypesTest() throws NoSuchFieldException {
 			}
 
 			Class<?> paramType = ConversionUtils.boxIfNeeded(method.getParameterTypes()[0]);
-			if (ConversionUtils.isIterableNonByteArrayType(paramType)) {
+			if (paramType.equals(Struct.class) || paramType.equals(Value.class)) {
+				/*
+				 * 1. there is a method for binding a Struct value, but because Struct
+				 * values cannot be written to table columns we will ignore it. 2. there
```

Comment:
What about using structs in the `WHERE` clause? Does that work? Do we support it?

Reply:
This class is specifically only testing the methods for writing to columns during creating a mutation to send to spanner, and as a result we don't support structs because columns in spanner cannot store structs. 

Separately, we also can't do equality comparisons on entire structs.